localization/pt/chain-of-responsability/README.md

```diff
@@ -0,0 +1,174 @@
+---
+title: Chain of responsibility
+category: Behavioral
+language: en
```

Comment:
Here should be the language code. See https://github.com/iluwatar/java-design-patterns/wiki/15.-Support-for-multiple-languages

Reply:
@iluwatar updated language pt in README.md